src/foam/nanos/controller/ApplicationController.js

```diff
@@ -28,18 +28,19 @@ foam.CLASS({
   documentation: 'FOAM Application Controller.',
 
   implements: [
-    'foam.nanos.client.Client',
-    'foam.nanos.controller.AppStyles'
+    'foam.box.Context',
+    'foam.nanos.controller.AppStyles',
```

Comment:
FOAM style guide says no trailing commas.

Reply:
Removed here but I'm sure I'm doing this in other places. This feels like something a tool should do for us :p